hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCLI.java

```diff
@@ -57,8 +55,14 @@ public void testMkdirRootNonExistentContainer() throws Exception {
 
     String nonExistentContainer = "nonexistent-" + UUID.randomUUID();
 
+    // Disable creation of new filesystem
+    final AbfsConfiguration conf = getConfiguration();
+    conf.setBoolean(AZURE_CREATE_REMOTE_FILESYSTEM_DURING_INITIALIZATION,
+        false);
+
     int result = fsShell.run(new String[] { "-mkdir",
-        ABFS_SCHEME + "://" + nonExistentContainer + "@" + account + "/" });
+        ABFS_SCHEME + "://" + nonExistentContainer + "@" + account +
+            "/testDirOnRoot" });
```

Comment:
Do we need this change? HADOOP-16138 looks addressing the issue of "hadoop fs mkdir /" on nonexistent container.

Reply:
Reverted the path back to container root.